spec/unit/azurerm_server_create_spec.rb

```diff
@@ -374,8 +374,66 @@
             :set_default_image_reference!)
     end
 
+    describe 'security_group_exist' do
+      module Azure
+        module ARM
+          class DummyClass < Azure::ResourceManagement::ARMInterface
+          end
+        end
+      end
+
+      before do
+        @dummy_class = Azure::ARM::DummyClass.new
+        @resource_group_name = 'rgrp-2'
+        @vnet_name = 'vnet-2'
+        @sec_grp_name = 'sec_grp_2'
+      end
+
+      it 'given security group exist under the given resource group and return true' do
```

Comment:
This should form a sentence like `it 'returns true if the given security group exists under the given resource group'`